src/app/zap-templates/common/ClustersHelper.js

```diff
@@ -621,11 +671,11 @@ Clusters.init = async function(context) {
   const promises = [
     Promise.all(loadTypes),
     loadEndpoints.call(context),
-    loadClusters.call(context),
-    loadCommands.call(context, packageId),
-    loadAttributes.call(context, packageId),
+    loadClusters.call(context, packageId, includeAll),
+    loadCommands.call(context, packageId, includeAll),
+    loadAttributes.call(context, packageId, includeAll),
     loadGlobalAttributes.call(context, packageId),
-    loadEvents.call(context, packageId),
+    loadEvents.call(context, packageId, includeAll),
```

Comment:
Any chance to have 2 methods instead of a `boolean` ? (e. g `loadClusters` / `loadAllClusters`) ?

Reply:
Yes, we could do that.

Reply:
Done.